docs/Reference/EthSigner-CLI.md

```diff
@@ -45,27 +67,25 @@ Endpoint to which received requests are forwarded.
 
 ### downstream-http-request-timeout
 
-Timeout period (in milliseconds) for downstream requests. Default is 5000. 
+Timeout period (in seconds) for downstream requests. Default is 5. 
```

Comment:
You may want to check with @jframe, but my understanding is milliseconds are now to be the standard time input unit

Reply:
Fixed